packages/browser/src/index.ts

```diff
@@ -69,5 +69,9 @@ export { makeBrowserOfflineTransport } from './transports/offline';
 export { browserProfilingIntegration } from './profiling/integration';
 export { spotlightBrowserIntegration } from './integrations/spotlight';
 export { browserSessionIntegration } from './integrations/browsersession';
+export {
+  featureFlagsIntegration,
+  type FeatureFlagsIntegration,
```

Comment:
Is exporting this type functionally necessary? If not, I would not export it.

Reply:
It's useful for typescript users who get the integration instance to call `setFlag`. With this code snippet from the JSDoc:
```
const flagsIntegration = Sentry.getClient()?.getIntegrationByName<FeatureFlagsIntegration>('FeatureFlags');
if (flagsIntegration) {
  flagsIntegration.setFlag('my-flag', true);
}
```
The type is needed to make the linter happy + expose `setFlag` through autocomplete.

Note to self to add a JSDoc for `setFlag`.